docs/Installation/Installation.md

```diff
@@ -327,77 +282,71 @@ yaml-cpp@develop~shared` (note that is a tilde and not a dash in front
-    run `./build`; the script will then ask you questions to guide you towards
-    the correct settings (see notes below for additional details).
-    Then compile Charm++.
-    The Charm++ build will be located in a new directory,
-    `CHARM_DIR/ARCH_OPTS`, whose name may (or may not) have some of the options
-    appended to the architecture.
-    In addition to the core `charm++` target, you will need to compile either
-    the `LIBS` target or the `everylb` target. This is needed so that we can
-    support the more sophisticated load balancers in SpECTRE executables.
-  * On macOS 10.12 it is necessary to patch the STL implementation. Insert
-    \code
-    #ifndef _MACH_PORT_T
-    #define _MACH_PORT_T
-    #include <sys/_types.h> /* __darwin_mach_port_t */
-    typedef __darwin_mach_port_t mach_port_t;
-    #include <pthread.h>
-    mach_port_t pthread_mach_thread_np(pthread_t);
-    #endif /* _MACH_PORT_T */
-    \endcode
-    into the front of
-    `/Library/Developer/CommandLineTools/usr/include/c++/v1/__threading_support`
-3.  Return to `SPECTRE_ROOT`, and create a build dir by running
-    `mkdir build && cd build`
-4.  Build SpECTRE with
-    `cmake -D CHARM_ROOT=CHARM_DIR/ARCH_OPTS SPECTRE_ROOT`
-    then
-    `make -jN`
-    to compile the code.
-5.  Run the tests with
-    `make test-executables && ctest`.
-
-**Notes**:
-* For more details on building Charm++, see the directions
-  [here](http://charm.cs.illinois.edu/manuals/html/charm++/A.html)
-  The correct target is `charm++` and, for a personal machine, the
-  correct target architecture is likely to be `multicore-linux-x86_64`
-  (or `multicore-darwin-x86_64` on macOS).
-  On an HPC system, the correct Charm++ target architecture depends on the
-  machine's inter-node communication architecture. We will be providing specific
-  instructions for various HPC systems.
-* Both Charm++ and SpECTRE must be compiled using the same compiler,
+## Building Charm++
+
+If you are not using a container, haven't installed Charm++ with Spack, or want
+to install Charm++ manually for other reasons, follow the installation
+instructions in the [Charm++ repository](https://github.com/UIUC-PPL/charm)
+and in their [documentation](https://charm.readthedocs.io/en/latest/quickstart.html#installing-charm).
+Here are a few notes:
+
+- Once you cloned the [Charm++ repository](https://github.com/UIUC-PPL/charm),
+  run `git checkout v6.10.2` to switch to a supported, stable release of
+  Charm++.
+- Choose the `LIBS` target to compile. This is needed so that we can support the
+  more sophisticated load balancers in SpECTRE executables.
+- On a personal machine the correct target architecture is likely
+  `multicore-linux-x86_64`, or `multicore-darwin-x86_64` on macOS. On an HPC
+  system the correct Charm++ target architecture depends on the machine's
+  inter-node communication architecture.
+- Both Charm++ and SpECTRE must be compiled using the same compiler,
```

Comment:
This is not try, the compilers just need to be ABI compatible. I only have a GCC build of Charm++ but use Clang for almost all spectre things.